src/citrine/resources/project.py

```diff
@@ -150,6 +152,13 @@ def generative_design_executions(self) -> GenerativeDesignExecutionCollection:
         """Return a collection representing all visible generative design executions."""
         return GenerativeDesignExecutionCollection(project_id=self.uid, session=self.session)
 
+    @property
+    def sample_design_space_executions(self) -> SampleDesignSpaceExecutionCollection:
```

Comment:
Since this requires a design space ID, it doesn't belong as a property on the `ProjectCollection`. It would fit on a `DesignSpace` object, similar to how we have [reports on predictors](https://github.com/CitrineInformatics/citrine-python/blob/main/src/citrine/informatics/predictors/graph_predictor.py#L120).